src/main/java/ch/njol/skript/variables/Variables.java

```diff
@@ -274,6 +274,23 @@ public static VariablesMap removeLocals(Event event) {
 	public static void setLocalVariables(Event event, Object map) {
 		localVariables.put(event, (VariablesMap) map);
 	}
+
+	/**
+	 * Creates a copy of the VariablesMap for local variables in an event.
+	 * This method should only be accessed in Bukkit's main thread.
```

Comment:
Due to #3817, it should be fine to access outside of the main thread. I just forgot to remove that Javadoc comment on the field, could you do that as well?